app/assets/javascripts/trips.js

```diff
@@ -0,0 +1,17 @@
+function showCurrentTrips(){
+  document.getElementById("current_trips").style.display = "block";
+  document.getElementById("past_trips").style.display = "none";
+  document.getElementById("all_trips").style.display = "none";
+}
+
+function showPastTrips(){
+  document.getElementById("past_trips").style.display = "block";
+  document.getElementById("current_trips").style.display = "none";
+  document.getElementById("all_trips").style.display = "none";
+}
+
+function showAllTrips(){
+  document.getElementById("all_trips").style.display = "block";
+  document.getElementById("past_trips").style.display = "none";
+  document.getElementById("current_trips").style.display = "none";
```

Comment:
If you are creating a function to hide all buttons on the page, and then just show one of the buttons, then this function can be re-used here.